scripts/linters/general_purpose_linter.py

```diff
@@ -583,6 +583,20 @@
         'message': 'Please use keys() instead.',
         'excluded_files': (),
         'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\.put_async\('),
+        'message': (
+            'Please use put() instead.'),
+        'excluded_files': (),
+        'excluded_dirs': ('core/storage/base_model/',)
```

Comment:
How about this, should we remove it too?

Reply:
Yes I think we should remove it and I have removed it. Thanks.